src/main/resources/templates/subscription.html

```diff
@@ -15,11 +15,16 @@
 
 <body>
     <div class="d-none" id="subscriptionTemplate" th:text="${subscriptionTemplate}"></div>
-    <div class="d-flex">
-        <div class="col-md-9 col-8">
+    <div class="d-flex flex-wrap">
+        <div class="col-md-4 col-6">
             <h3>Eiffel Intelligence Subscription Handling</h3>
         </div>
-        <div class="col-md-3 col-4 d-flex justify-content-end">
+        <div class="col-md-6 col-12">
```

Comment:
order-last order-md-1, is missing in class.

Reply:
Done